src/test/java/api/support/fixtures/ItemsFixture.java

```diff
@@ -115,6 +116,37 @@ public InventoryItemResource basedUponSmallAngryPlanet()
     return basedUponSmallAngryPlanet(identity());
   }
 
+  public InventoryItemResource basedUponSmallAngryPlanet(
+    boolean applyHoldingRecord,
```

Comment:
I would prefer that there wasn't branching within the tests if possible (at the risk of being hypocritical).

I personally find it harder to understand what is going on when the behaviour is dependent upon a boolean.

As these examples seem to only be used within the context of patron notices, could this be moved to a more specific class? 

Maybe with two new examples, which explicitly describe the two or three scenarios (e.g. holdings properties present or item properties present)?

Reply:
I will think about how to exclude an explicit branching in the current implementation.